src/cider/nrepl/middleware/debug.clj

```diff
@@ -297,19 +297,21 @@ this map (identified by a key), and will `dissoc` it afterwards."}
                               last :stacktrace)}]}))
 
 (def debug-commands
-  {"c" :continue
-   "C" :Continue
-   "e" :eval
-   "h" :here
-   "i" :in
-   "j" :inject
-   "l" :locals
-   "n" :next
-   "o" :out
-   "p" :inspect
-   "q" :quit
-   "s" :stacktrace
-   "t" :trace})
+  "An unsorted set of commands supported by the debugger."
+  #{:continue
+    :Continue
```

Comment:
We should use a proper name here - the current one was just to convey the relationship with `C`.

Reply:
`:continue-all` and `:inspect-prompt`? Or maybe `:inspect-current` for `p` 
Would that also call for a command name -> display name mapping on the client side? Right now it's just using the plain command names as overlay titles.

Reply:
You read my mind. I was just writing about some display name mapping on the other ticket. :-)

> :continue-all

👍 

For inspect - let's have the default command inspect the current value and the other to be `inspect-prompt`.